src/types.ts

```diff
@@ -0,0 +1,5 @@
+import createApp from ".";
+import { AppBridgeState } from "./app";
+
+export type App = ReturnType<typeof createApp>;
```

Comment:
Not a fan of this, explicitly typing return type by `function createApp(): Type` is making breaking api harder. Now it's possible to change function signature and no one will know until consumers start complaining about breaking changes.